src/runtime/policy/policy.ts

```diff
@@ -128,8 +131,8 @@ export class PolicyTarget {
     const fields = node.fields.map(field => PolicyField.fromAstNode(field, buildAnnotationRefs));
 
     // Process annotations.
-    // TODO(b/157605585): Validate maxAge annotation.
     const allAnnotations = buildAnnotationRefs(node.annotationRefs);
+    let maxAge: Ttl | null = null;
```

Comment:
in recipes we consider TTL unlimited, unless specified otherwise. but in policies i think we consider TTL to be zero, unless specified. so probably you should use Ttl.none() here as a default?

Reply:
Good idea, done.